include/swift/AST/DiagnosticsSema.def

```diff
@@ -2002,6 +2002,9 @@ ERROR(inheritance_from_cf_class,none,
 ERROR(inheritance_from_objc_runtime_visible_class,none,
       "cannot inherit from class %0 because it is only visible via the "
       "Objective-C runtime", (Identifier))
+ERROR(inheritance_from_class_with_no_designated_initializers,none,
+	  "cannot inherit from class %0 because it has no designated initializers",
```

Comment:
Drive-by nit: spaces, not tabs, are used in this file; please align this line as shown above.